modules/ROOT/pages/setting-environment-variables.adoc

```diff
@@ -0,0 +1,28 @@
+= Setting Environment Variables
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+:keywords: anypoint studio, esb, environment variables, system variables, system properties
+
+If you're deploying multiple applications through a Shared Resources structure, then you shouldn't set any properties in the project properties files, as there might potentially be conflicts between the various apps that share a domain. Instead, you can set environment variables over the scope of the deployed app, its domain and other apps under that domain.
+
+The steps below explain how to set these environment variables in Studio. Note that these variables will only be used when deploying locally through Studio, when deploying your applications to Mule Standalone, you must provide these variables through the command line instead.
```

Comment:
Re "to set these environment variables": as opposed to some other environment variables? IF not, maybe "to set environment variables" without "these".

"will only be used" -> "are only used"

"when deploying locally" -> "when you deploy locally" maybe?

Reply:
is it "Mule Standalone" or "Mule standalone"?

Reply:
"when deploying locally" -> "when you deploy locally" maybe?